### DISCUSSION TITLE
Eligibleto use test_x to fine tune model?

### DISCUSSION THREAD

**Discussion by Chaodong** [PaddlePaddle/PaddleSpatial#187] [BaiduKDDCup2022] [answered]
Is it Eligible to use test_x to fine tune my model?

**Comment by ramber1836** [comment, marked as answer]
Yes, you can use test_x to fine-tune your model.

**Reply by hongyingyue** [reply]
@ramber1836 

Hello, so can we use all the test_x to fine tune the model, or only the data before test day to fine tune it?

Because there is some risk to leak the future information during fine tune.

**Reply by Chaodong** [reply]
I think only the current text_x, since the test cases are ordered by time, otherwise there will be information leaking. Waiting for official confirmation!

**Reply by ramber1836** [reply]
We will update the toy data (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_test_toy) and only a few more test samples will be added.

**Reply by Chaodong** [reply]
> We will update the toy data (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_test_toy) and only a few more test samples will be added.
Actually my question means that can we use text_x online to finetune model?

**Reply by Chaodong** [reply]
The original question means to use text_x online (during submission) to finetune model.

**Reply by ramber1836** [reply]
You can use the test_x in the online test to fine-tune your model.
